src/PAModel/CanvasDocument.cs

```diff
@@ -445,6 +445,23 @@ private void StabilizeAssetFilePaths()
                 if (!_assetFiles.TryGetValue(assetFilePath, out var fileEntry))
                     continue;
 
+                if (_entropy.LocalResourceFileNames.ContainsKey(resource.Name))
+                {
+                    int i = 1;
+                    var newResourceName = resource.Name + '_' + i;
+                    while (_entropy.LocalResourceFileNames.ContainsKey(newResourceName))
```

Comment:
@lesaltzm will this handle that scenario elegantly when the after renaming the new name becomes image_1.jpg but file with image_1.jpg exists already. In that case it should go another round of renaming which should make the newName = image_1_1.jpg

Reply:
Right, so, in that case, if we had `image`, `Image`, and `image_1`, we wind up with `image` `Image_1` and `image_1_1`

It's not elegant, per-se. We could do this as two passes, where we first populate the names, and then do renames, that might be better.

Reply:
ok was just checking if it handles this, and not try to persist two files with image_1 names which could break unpacking

Reply:
Ok, pushed a better rename, with test. 
Now we get 
`image`, `Image_2` and `image_1`